### PR TITLE
Feature/mvvm refactor

### PR DESCRIPTION
**Pull request by NathanDVT**

### CODE REVIEW

**Review by AkuaQ — APPROVED** [review]

File: realwork/Podfile [hunk 1]
```diff
@@ -6,8 +6,8 @@ target 'realwork' do
   use_frameworks!
 
   # Pods for realwork
-  pod 'NLibrary', :git => 'https://github.com/NathanDVT/NLibrary.git', :branch => 'development'
-#  pod 'NLibrary', :path => '/Users/nngobale/Documents/Training 2020/Pod_Folder/NLibrary/NLibrary'
+#  pod 'NLibrary', :git => 'https://github.com/NathanDVT/NLibrary.git', :branch => 'development'
+  pod 'NLibrary', :path => '/Users/nngobale/Documents/Training 2020/Pod_Folder/NLibrary/NLibrary'
```
Comment (Tiewhan) [inline realwork/Podfile]: This points to a local file. It needs to point to a repository

File: realwork/Podfile.lock [hunk 2]
```diff
@@ -2,20 +2,15 @@ PODS:
...
-  NLibrary:
-    :commit: 149836493a96bc7777d9b3c4873f942e25b8d921
-    :git: https://github.com/NathanDVT/NLibrary.git
+    :path: "/Users/nngobale/Documents/Training 2020/Pod_Folder/NLibrary/NLibrary"
 
 SPEC CHECKSUMS:
   NLibrary: ec2ddce5738362f9b660f27b78723df7e5fe7766
 
-PODFILE CHECKSUM: 4770deebebd51abf54033475bffbabadc656aa4f
+PODFILE CHECKSUM: a39daa594c047f1ff898c3094e7d90160b7441d8
```
Comment (Tiewhan) [inline realwork/Podfile.lock]: All of this is effected by the change to the pod file. Run pod update with the pod file pointing to a git repo

File: realwork/realwork/ArtistMediaTableViewController.swift [hunk 3]
```diff
@@ -14,6 +14,7 @@ class ArtistMediaTableViewController: UITableViewController {
     var listOfArtistCollections = [Collection]() {
         didSet {
             DispatchQueue.main.async {
+                print ("reset values")
```
Comment (Tiewhan) [inline realwork/realwork/ArtistMediaTableViewController.swift]: Another Print

**Review by Tiewhan — CHANGES_REQUESTED** [review]
The Pod file has to be changed for this to be a proper working pull request

**Review by AkuaQ — APPROVED** [review]

**Review by Tiewhan — APPROVED** [review]

**Pull request merged**